examples/language/llama/benchmark.py

```diff
@@ -334,8 +365,12 @@ def empty_init():
                     return_loss=True,
                 )
                 loss = outputs["loss"]
-                if dist.get_rank() == dist.get_world_size() - 1:
-                    print(f"Step {step} loss: {loss}")
+                if args.pp_style == "zbv":
+                    if dist.get_rank() == 0:
```

Comment:
`coordinator.is_master()`

Reply:
> `coordinator.is_master()`

Fixed in https://github.com/hpcaitech/ColossalAI/pull/6083/commits/5f0924361de4e87f05cbf8aadf9fbd698873a53d;